README.md

```diff
@@ -63,14 +62,19 @@ For more, see [responsibilities](#responsibilities).
 + This must be a separate company from Founders & Coders CIC,
 so that F&C can focus solely on the ***mission***;
 Providing ***great technical learning***, ***free*** and ***open source***.
-+ The company will and should sit in on F&C meetings but *not*
-have the power to dictate what happens in the school.
++ The company will and should participate in F&C meetings but _not_
+have the power to _dictate_ what happens in the school.
```

Comment:
please *stop* using underscores for italics. some markdown parsers don't support them. we have ended up with underscores in the browser view on several occasions.